pkg/kubelet/server.go

```diff
@@ -148,7 +202,8 @@ func (s *Server) InstallDefaultHandlers() {
 		Operation("getPods"))
 	s.restfulCont.Add(ws)
 
-	s.restfulCont.Handle("/stats/", &httpHandler{f: s.handleStats})
+	s.restfulCont.HandleWithFilter("/stats/", &httpHandler{f: s.handleStats})
```

Comment:
> So anything that slips and is registered via Handle is exposed?

That's really scary.  Can you change the type to an interface that only has `HandleWithFilter` as an option so that its really hard to do?